src/main/java/com/xiaomi/infra/pegasus/example/ExampleSample.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2020, Xiaomi, Inc.  All rights reserved.
+// This source code is licensed under the Apache License Version 2.0, which
+// can be found in the LICENSE file in the root directory of this source tree.
+package com.xiaomi.infra.pegasus.example;
+
+import com.xiaomi.infra.pegasus.client.*;
+import java.io.UnsupportedEncodingException;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.lang3.RandomStringUtils;
+
+/** Example sample */
+public class ExampleSample {
```

Comment:
I think this example is too naive to show any best practice of the SDK. The Full-Scan example is good. But this example I'm afraid has no much value.

A better example must show the advanced cases that this SDK can handle. The hbase-client example:

https://github.com/apache/hbase/tree/master/hbase-examples/src/main/java/org/apache/hadoop/hbase/client/example

I suggest you can remove this sample from this PR. And we can go on merge the full-scan sample.

Reply:
Ok, I will do these things later, but there is a problem. I think that these examples should not be included when packaged into a jar package. They should not be included with the core code. What do you think of putting it outside?

Reply:
Yes. I'm aware of this too. I think it's fine to let it be packaged for now, like HBase. 
Users are not able to directly use the example.